src/features/surveys/models/SurveyDataModel.ts

```diff
@@ -120,6 +120,16 @@ export default class SurveyDataModel extends ModelBase {
     }
   }
 
+  get surveyIsEmpty(): boolean {
+    const { data } = this.getData();
+
+    if (!data) {
+      return true;
+    }
```

Comment:
This is risky. It means that while the survey is loading, it's going to appear to be empty. Might be fine, but lets be prepared for bugs here.